crypto/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class CryptoException(Exception):
```

Comment:
I usually like to prefix exceptions with the same name. So for example you could do `ArkCryptoException` and `ArkSerializerException` or something like that. That's just my opinion, as I like to know from where exception comes from straight from the name rather than the full path of the exception. Up to you, can easily leave it as it is right now.